src/tests/TableWithComponents.test.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, fireEvent } from '@testing-library/svelte';
+
+import Table from '../mocks/TableWithComponents.svelte';
+
+describe('SvelteTable - custom components', () => {
+  it('should render a table custom head and cell slots', () => {
+    const { getByText } = render(Table, { empty: false });
+
+    expect(getByText(/navn/gi)).toBeInTheDocument();
```

Comment:
Nitpick: Consider using english here as well

Reply:
it was to test if i can set in a custom component inside of a slot that isn't the same as the original one, i could probably add like a data-attribute or something and test that instead, if that makes any difference for the test purposes

Reply:
updated